5_databases/week1/homework.sql

```diff
@@ -0,0 +1,52 @@
+USE hyf_lesson1;
+
+-- 1 Find out how many tasks are in the task table
+
+SELECT COUNT (title) FROM task 
```

Comment:
No space between CONNT and `(title)`. And just use `*` instead of `title` because you dont have to specify a certain field here